### ISSUE TITLE
Hint text

### ISSUE THREAD

**Issue by gavinwye** [hmrc/design-patterns#97]
Light gray or black text to help users complete forms.

**Comment by stevenaproctor** [comment]
I think the grey colour reduces the content's importance and is easily overlooked. Plus, the colour contrast of the grey text is 4.56:1, which is just over 4.5:1 required for WCAG AA. Black text on white colour contrast is 21:1. The problem will be differentiating the hint text from the form label if the label is standard text too.

**Comment by gavinwye** [comment]
@stevenaproctor do you have any examples from user research that backs up the fact that grey content is easily overlooked?

**Comment by stevenaproctor** [comment]
@gavinwye Not specifically. We have changed a lot of our explanatory text to be regular paragraphs before input fields rather than using the grey hint text. The only place we tend to have that is next to date fields.

But we have seen people not read content on the screen no matter the colour, size or format. There is a lot of evidence that people do not read content, do not read the entire page, or miss content when they are skimming. The lighter appearance of the content would make it easier to not read or miss.

**Comment by chris-harding** [comment]
Myself and Kate Phipps have scheduled some usability testing of help text being used in the ITR service. This testing will cover colour, position and the content.

Our user testing scope is across a selection of government services, with the hope of learning how different approaches have worked for different users and services. After collating this research data we will analyse and then propose a draft design for help text, which we will then perform some separate usability testing on.

We will report back with our findings.

**Comment by chris-harding** [comment]
**Checklist for this issue**

1. Examine a number of services (across all DDCs) to identify the different situations that grey and black text is currently being used
2. Ask these services for the user research and / or evidence that resulted in them implementing the hint / help text in the way it has
3. Conduct fresh user research around hint / help text to try and fill any gaps that exist
4. Incorporate questions that have hint / help text into a prototype and then user test these examples
5. Gather opinions from a sample of designers across the various centres
6. Analyse research findings to identify trends, new user needs and a potential design pattern to be used moving forwards
7. Test the proposed design pattern with users
8. Present final findings and the proposed design pattern to design community

These will be updated as progress is made.

**Comment by stevenaproctor** [comment]
@gavinwye Component? It is part of elements.

**Comment by gavinwye** [comment]
@stevenaproctor yes this is part of elements.

**Comment by philsherry** [comment]
I've done UR sessions with visually-impaired people who had real trouble reading the grey hint text. Feedback was along the lines of make the labels bold and the hint text the same as a body text.

**Comment by stevenaproctor** [comment]
Closed as it is part of the GOV.UK design system backlog.

**Comment by jennifer-hodgson** [comment]
Use of hint text is now covered by[ this issue relating to addition guidance for designing question pages](https://github.com/hmrc/design-patterns/issues/156).
